wolfcrypt/src/aes.c

```diff
@@ -9864,10 +9946,17 @@ static void roll_auth(Aes* aes, const byte* in, word32 inSz, byte* out)
         xorbuf(out + authLenSz, in, inSz);
         inSz = 0;
     }
-    wc_AesEncrypt(aes, out, out);
+    ret = wc_AesEncrypt(aes, out, out);
```

Comment:
Can you re-write this with single `return` point? Would be much cleaner. You already have `ret` and it is always `0` on success.

Reply:
done.